doc/uftrace-recv.md

```diff
@@ -27,6 +27,36 @@ Data will be sent using `uftrace-record` with -H/\--host option.
 :   Run given (shell) command as soon as receive data.  For example, one can run "uftrace replay" for received data.
 
 
+EXAMPLE
+=======
+ The uftrace recv command should be run before sending data by record command.
+
+    # HOST 
+    $ uftrace recv -d recv_data --port 1234
+
+ Above command starts a server with port by given (default `8090`.) to receive data from remote client. 
```

Comment:
Sorry for nitpicking, but please remove '.' since it's not an end of a sentence.